README.md

```diff
@@ -244,17 +334,21 @@ functions.
     @transform        mutate           Select (?)
     @by                                GroupBy
     groupby           group_by
```

Comment:
While we're at it, LINQ's `GroupBy` sounds quite similar to our `groupby` as it returns an iterable of groups. It also has methods to apply a combining step directly which are more like `@by`, but better mention both IMO.
See e.g. https://killalldefects.com/2020/01/21/dotnet-linq-groupby-in-depth/

```suggestion
    groupby           group_by         GroupBy
```

(TBH I think `by` could go away as people seem to be OK without it e.g. in dplyr.)